slices/base-files.yaml

```diff
@@ -25,6 +25,8 @@ slices:
     contents:
       /lib:
       /usr/lib/:
+      /lib64:
+      /usr/lib64/:
```

Comment:
These seem to be available only in a handful or arch. Maybe we should specify them.

Reply:
Y for sure. Just wanted to make sure those archs weren't being updated upstream. Should be good now